optimize/self-managed/optimize-deployment/migration-update/3.9-to-3.10-preview.md

```diff
@@ -0,0 +1,23 @@
+---
+id: 3.9-to-3.10-preview
+title: "Update notes (3.9.x to 3.10-preview)"
```

Comment:
we have dropped preview releases remember? This should be an upgrade to 3.10 directly